### PR TITLE
Adds the ice dispenser and puts it in bars

### PR DESCRIPTION
**Pull request by Carbadox**
## About the PR 
Adds an ice dispenser for bar tenders

## Why's this needed? 
Much more formal source of ice than freezing water in a heater/cooler unit

## Changelog

```changelog
(u)Carbadox and Gannets
(+)Added an ice dispenser to most bars.
```

### CODE REVIEW

**Comment by Azrun** [comment]
~Please convert to standard ice maker and have it have a chance of breaking glass containers which will then cause the dispenser to produce glass shards on occasion.~

Nice sprite and gives cozy vibes.

**Comment by Carbadox** [comment]
I've decided to move their placements around because the way I originally put them makes them stick out and look really bad even though its a good sprite.

File: maps/kondaru.dmm [hunk 1]
```diff
@@ -52211,15 +52211,11 @@
 /obj/disposalpipe/junction/left/south,
 /turf/simulated/floor/plating,
 /area/station/maintenance/south)
-"eFl" = (
-/turf/simulated/wall/auto/reinforced/supernorn/yellow,
-/area/station/mining/magnet)
+/obj/stool/bar,
```
Comment (pali6) [inline maps/kondaru.dmm:52214]: This is invalid syntax, no tile identifier precedes it.

Comment (Carbadox) [reply maps/kondaru.dmm:52214]: i think I messed up solving a merge conflict. I'll try and fix this

Comment (Carbadox) [reply maps/kondaru.dmm]: should be fixed

File: code/obj/item_dispensers.dm [hunk 2]
```diff
@@ -99,9 +99,17 @@
...
-	amount = 7 
+	amount = 7
 
 	attack_hand(mob/user as mob)
 		if (!src.cant_withdraw && src.amount >= 1)
 			playsound(src.loc, "sound/machines/printer_dotmatrix.ogg", 25, 1)
-		..()
\ No newline at end of file
+		..()
+/obj/item_dispenser/icedispenser
```
Comment (pali6) [inline code/obj/item_dispensers.dm:108]:
```suggestion

/obj/item_dispenser/icedispenser
```

Leaving an empty line makes it more readable.

**Review by pali6 — APPROVED** [review]

**Review by flrsh — CHANGES_REQUESTED** [review]
Please keep the television in the Cog1 bar and the credit scanner in the Destiny bar, somewhere.

**Comment by Carbadox** [comment]
Is it fine if I move the TV in cog1 bar to a different location? I don't see alot of people using it and Im having difficulties finding a spot to place the ice dispenser that both looks good and is easily accessible. Also, there's a credit transfer device on the front bar desk on Destiny, so only 5 tiles away from the one I removed. Is that also fine?

**Comment by flrsh** [comment]
Different location for the TV on Cog1 sounds good, I'd prefer the credit transfer device to stay there on Destiny since it's hall facing, but also there's a PR that overhauls Destiny so maybe my comment won't be relevant anymore.

**Review by flrsh — APPROVED** [review]

**Pull request merged**